client/overview/task-list/dispute-tasks.ts

```diff
@@ -28,12 +29,10 @@ const isDueWithin = ( dispute: CachedDispute, days: number ) => {
 	if ( dispute.due_by === '' ) {
 		return false;
 	}
-	const now = moment();
-	const dueBy = moment( dispute.due_by );
-	return (
-		dueBy.diff( now, 'hours' ) > 0 &&
-		dueBy.diff( now, 'hours' ) <= 24 * days
-	);
+	// Get current time in UTC.
+	const now = moment().utc();
+	const dueBy = moment.utc( dispute.due_by );
+	return dueBy.diff( now, 'hours' ) > 0 && dueBy.diff( now, 'days' ) <= days;
```

Comment:
Comparisons are handled in UTC time for more robust logic consistent with PHP logic in this PR.